app/assets/stylesheets/active_admin/mixins/_variables.scss

```diff
@@ -40,3 +40,12 @@ $text-input-horizontal-padding: 10px !default;
 $text-input-total-padding: $text-input-horizontal-padding * 2 + $border-width * 2;
 
 $blank-slate-border: 1px dashed #DADADA !default;
+
+// Sidebar Forms
+$sidebar-inner-content-width: $sidebar-width - ($section-padding * 2);
+
+// Filter Forms
+$filter-field-seperator-width: 12px !default;
+$side-by-side-filter-input-width: ($sidebar-inner-content-width * 0.5) - ($text-input-horizontal-padding * 2) - $filter-field-seperator-width;
+$side-by-side-filter-select-width: ($sidebar-inner-content-width * 0.5) - $filter-field-seperator-width;
+$date-range-filter-input-width: ($sidebar-inner-content-width * 0.5) - ($filter-field-seperator-width * 0.5);
```

Comment:
The `$filter-field-seperator-width` variable is now marked `!default` here whereas in its original location it wasn't. If we are moving all the form based variables (note: include `$text-input-total-padding` as well) I think we should just mark these all with `!default` otherwise they can't be overridden.

Reply:
Hmm no, I guess not since we can always do so later if needed. We already have another calculated variable here so it should be fine. Thanks!

Reply:
I think the current changeset respects the previous criteria of only marking variables set to a literal as `!default`, not variables derived from other variables. Not sure if that's right or not, but this change keeps that consistent.